neqo-http3/src/features/extended_connect/webtransport_session.rs

```diff
@@ -43,8 +44,8 @@ pub struct WebTransportSession {
     state: SessionState,
     frame_reader: FrameReader,
     events: Box<dyn ExtendedConnectEvents>,
-    send_streams: BTreeSet<StreamId>,
-    recv_streams: BTreeSet<StreamId>,
+    send_streams: HashSet<StreamId>,
```

Comment:
What is the benefit of a `HashSet` over a BTreeSet` here?

Reply:
I was assuming `HashSet` to be faster?

Reply:
I think we should only make these changes if we have proof that they are more performant.

Connecting this to the discussion above, might ordering (i.e. thus `BTreeSet`) be relevant here as well?

Reply:
Fair point about only changing this after benchmarking.